src/ox_modules/module-utils.js

```diff
@@ -42,4 +44,64 @@ export default class UtilsModule extends OxygenModule {
         utils.assertArgumentNumberNonNegative(ms, 'ms');
         deasync.sleep(ms);
     }
+
+    /**
+     * @summary Reads data from csv file
+     * @function readCsv
+     * @param {String} filePath - Absolute path to file
+     * @param {Object} options - Options (https://csv.js.org/parse/options/)
```

Comment:
If this argument is optional, then it should be marked as such `=`

Reply:
fixed